lib/braintree/util.js

```diff
@@ -261,7 +261,7 @@ class Util {
     }
 
     if (invalidKeys.length > 0) {
-      return exceptions.InvalidKeysError(`These keys are invalid: ${invalidKeys.join(', ')}`); // eslint-disable-line new-cap
+      return exceptions.InvalidKeysError(`These custom fields are invalid: ${invalidKeys.join(', ')}`); // eslint-disable-line new-cap
```

Comment:
The phrase `custom fields` could be confused for https://articles.braintreepayments.com/control-panel/custom-fields

how about:

```suggestion
      return exceptions.InvalidKeysError(`These properties are invalid: ${invalidKeys.join(', ')}`); // eslint-disable-line new-cap
```

Reply:
I think that's nice!
I was building the following object and it didn't work, so yes I was using an invalid prop!
Thanks
` 
const saleRequest = {
     amount: '...',
     paymentMethodNonce: '...',
     orderId: '...',
     options: {
         submitForSettlement: true,
         paypal: {
             description: '...'
         }
     }
 }`